app/controllers/employments_controller.rb

```diff
@@ -48,11 +12,7 @@ def create
   def creation_service
     @creation_service ||= Creators::EmploymentsCreator.call(
       assessment_id: params[:assessment_id],
-      employments_attributes: input[:employment_income],
+      employments_attributes: JSON.parse(request.raw_post, symbolize_names: true),
```

Comment:
As discussed for parity with other controllers and creator services
```suggestion
      employments_params: request.raw_post
```